tests/RegisterAddres.test.js

```diff
@@ -0,0 +1,8 @@
+const {
+    ApiPromise,
+    WsProvider,
+    Keyring
+} = require("@polkadot/api");
+
+it('RegisterAddress should work', () => {
```

Comment:
@atodorov there is a `balances.Withdraw` event associated with extrinsics, this is the transaction fee. See https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fqa.creditcoin.network#/explorer/query/0xa9ef1a0a6be6470f7291c04ed454dff2738fad1116b9a28baa4d12148da79b8b.